e2e-tests/cypress/support/e2e.js

```diff
@@ -19,3 +19,5 @@ import './commands';
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
 import 'cypress-axe';
+import { slowCypressDown } from 'cypress-slow-down';
+slowCypressDown();
```

Comment:
Do we need this in the old tests? Best to only add it for the tests which need it.

Reply:
Will uninstall it